cli/examples/project/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM atk-base:latest
+
+# Any software dependencies you may need
+# For example: The python library "shapely",
+# which is in "requirements.txt" requires 'libgeos-dev'
+RUN  apt-get -qq update -y \
+  && apt-get install libgeos-dev -y
+
+# Copy contents of your project into the docker container
+COPY . /atk-example-project
```

Comment:
maybe change these to just /atk-example so that the path name matches the container name?

Reply:
Sure